.github/workflows/serp-ai_02_e2e_staging.yml

```
@@ -0,0 +1,60 @@
+name: e2e_staging_serp_ai
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)